src/Microsoft.ManagedZLib/benchmarks/ManagedZLibBenchmarks.cs

```diff
@@ -62,15 +63,25 @@ public void DecompressManaged()
     {
         CompressedFile!.CompressedDataStream.Position = 0;
         _outputStream!.Position = 0;
-
-
         DeflateStream decompressor = new DeflateStream(CompressedFile.CompressedDataStream, CompressionMode.Decompress);
         decompressor.CopyTo(_outputStream);
     }
 
     public class ProgramRun
     {
-        public static void Main(string[] args) => BenchmarkSwitcher.FromAssembly(typeof(ProgramRun).Assembly).Run(args);
+        public static void Main(string[] args)
+        {
+            var job = Job.Default
+                .WithWarmupCount(1) // 1 warmup is enough for our purpose
```

Comment:
It's enough to ensure that all methods are promoted to tier 1?

Reply:
Adam Sitnik suggested the change because it's the recommended configuration from the perf repo: [(Link to source)](https://github.com/dotnet/performance/blob/main/src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs#L34). He said the vast majority of the methods get promoted during Pilot stage.